### DISCUSSION TITLE
What are the different ways that the eye-gaze could stop functioning?

### DISCUSSION THREAD

**Discussion by jakemcivor** [ATMakersOrg/EyeAlert#5] [Q&A]
What are the different ways that the eye-gaze could stop functioning?
 - camera turns off (No IR light)
 -

**Comment by ImproveAbility** [comment]
- Communication between camera and software get confused.  I have seen this fail where the camera is disconnected (no IR) and it has also failed where camera stays on, but user loses control.
- Pop up comes up over eye gaze software that disables users ability to access items.  I have seen this happen with the low battery warning in windows as well as with video ads.
- Computer crashes.
- Computer freezes.

**Comment by NachumLehman** [comment]
Caregivers kicking the device out of position needs an honorable mention:(

**Comment by ImproveAbility** [comment]
client moving out of position over time.

**Comment by JayToltTech** [comment]
* Change in lighting (e.g. sun goes lower/higher in the sky, new reflections, a lightbulb is turned on/off)

* Change in pALS condition (e.g. tiredness increases ptosis, anti secretion medication dries out the eyes, muscle relaxants interfere with eye movement)

**Comment by JayToltTech** [comment]
Background update improperly completes, leaving the eye tracker software stack in a non functional state

Background update corrupts/removes screen position data or calibration data

Operating system crash corrupts files such as screen position or calibration data due to a corrupted file write

**Comment by JayToltTech** [comment]
USB hub the eye tracker is attached to enters a nonfunctional state, perhaps due to an electrostatic shock to the device from a nearby caregiver.
